srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config php"
+makedepends="libsndfile-devel libX11-devel libglvnd-devel lv2 cairo-devel ladspa-sdk jack-devel"
+short_desc="Collection of free plugins compatible with LADSPA, LV2 and LinuxVST"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="LGPL-3.0-or-later"
+homepage="https://lsp-plug.in/"
+distfiles="https://github.com/sadko4u/lsp-plugins/archive/refs/tags/${version}.tar.gz"
+checksum=9cf43257729093c240375b3640b1514dff34b092b83b54a5ee68d7e8565c8f80
+
+_arch=$(echo $XBPS_TARGET_MACHINE | sed "s/-musl$//")
+case "$_arch" in
+	i686) export BUILD_PROFILE=i586 ;;
+	*)    export BUILD_PROFILE=${_arch} ;;
+esac
+
+# https://github.com/sadko4u/lsp-plugins/issues/191#issuecomment-812847630
+CXXFLAGS="$CXXFLAGS -std=c++98"
```

Comment:
The issue is straightforward, so I don't think you have to link it, just mention that lsp-plugins needs C++98+.

`CXXFLAGS="-std=c++98"` is enough. After the template is imported, the value of default CXXFLAGGS is merged with CXXFLAGS from the template.

Reply:
> The issue is straightforward, so I don't think you have to link it, just mention that lsp-plugins needs C++98+.
> 
> `CXXFLAGS="-std=c++98"` is enough. After the template is imported, the value of default CXXFLAGGS is merged with CXXFLAGS from the template.

See https://github.com/void-linux/void-packages/pull/31664#discussion_r659215151